distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,359 @@
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
+
+|`${ddf-branding}.catalog.data.Metacard` Constant
+|Attribute Name
+|Attribute Format
+|Description
+
+|`CONTENT_TYPE`
+|`metadata-content-type`
+|STRING
+|Attribute name accessing for the metadata content type of a Metacard.
+
+|`CONTENT_TYPE_VERSION`
+|`metadata-content-type-version`
+|STRING
+|Attribute name for the version of the metadata content accessing type of a Metacard.
+
+|`CREATED`
+|`created`
+|DATE
+|Attribute name for accessing the date/time *this Metacard* was created.
+
+|`EFFECTIVE`
+|`effective`
+|DATE
+|Attribute name for accessing the date/time of *the product* represented by the Metacard.
+
+|`EXPIRATION`
+|`expiration`
+|DATE
+|Attribute name for accessing the date/time the Metacard is no longer valid and could be removed.
+
+|`GEOGRAPHY`
+|`location`
+|GEOMETRY
+|Attribute name for accessing the location for this Metacard.
+
+|`ID`
+|`id`
+|STRING
+|Attribute name for accessing the ID of the Metacard.
+
+|`METADATA`
+|`metadata`
+|XML
+|Attribute name for accessing the XML metadata for this Metacard.
+
+|`MODIFIED`
+|`modified`
+|DATE
+|Attribute name for accessing the date/time this Metacard was last modified.
+
+|`RESOURCE_SIZE`
+|`resource-size`
+|STRING
+|Attribute name for accessing the size in bytes of the product this Metacard represents.
+
+|`RESOURCE_URI`
+|`resource-uri`
+|STRING
+|Attribute name for accessing the URI reference to the product this Metacard represents.
+
+|`TARGET_NAMESPACE`
+|`metadata-target-namespace`
+|STRING
+|Attribute name for the target namespace of the accessing metadata content type of a Metacard.
+
+|`THUMBNAIL`
+|`thumbnail`
+|BINARY
+|Attribute name for accessing the thumbnail image of the product this Metacard represents. The thumbnail must be of MIME Type `image/jpeg` and be less than 128 kilobytes. 
+
+|`TITLE`
+|`title`
+|STRING
+|Attribute name for accessing the title of the Metacard.
+
+|===
+
+[NOTE]
+====
+It is highly recommended when referencing a default attribute name to use the `ddf.catalog.data.Metacard` constants whenever possible.
+====
+
+[WARNING]
+====
+Every Source should at the very least return an ID attribute according to Catalog API.
+Other fields might or might not be applicable, but a unique ID must be returned by a <<_sources,Source>>.
+====
+
+===== Extensible Metacards
+
+Metacard extensibility is achieved by creating a new `MetacardType` that supports attributes in addition to the required attributes listed above.
+
+Required attributes must be the base of all extensible metacard types. 
+
+[WARNING]
+====
+Not all Catalog Providers support extensible metacards.
+Nevertheless, each Catalog Provider should at least have support for the default `MetacardType`; i.e., it should be able to store and query on the attributes and attribute formats specified by the default metacard type.
```

Comment:
Corollary: Catalog providers are neither expected nor required to store attributes that are _not_ in a given metacard's type.